arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -151,28 +151,30 @@ static arangodb::Result getReadLockId(network::ConnectionPool* pool,
   options.timeout = network::Timeout(timeout);
   options.skipScheduler = true; // hack to speed up future.get()
   
-  auto res = network::sendRequest(pool, endpoint, fuerte::RestVerb::Get,
+  auto response = network::sendRequest(pool, endpoint, fuerte::RestVerb::Get,
                                   REPL_HOLD_READ_LOCK,
                                   VPackBuffer<uint8_t>(), options)
                  .get();
+  auto res = response.combinedResult();
 
-  if (res.ok() && res.response->statusCode() == fuerte::StatusOK) {
-    auto const idSlice = res.response->slice();
+  if (res.ok()) {
```

Comment:
Previously we allowed only `200`. Now its `2xx`. Does this break something?

Reply:
As far as I can tell, the only reason for the more restricted check here was that it was the only 2xx status encountered, not that the others must be excluded.